yarn-project/bb-prover/src/bb/cli.ts

```diff
@@ -35,7 +35,7 @@ export function getProgram(log: LogFn): Command {
     )
     .requiredOption('-b, --bb-path <string>', 'The path to the BB binary', BB_BINARY_PATH)
     .requiredOption('-c, --circuit <string>', 'The name of a protocol circuit')
-    .requiredOption('-f, --flavor <string>', 'The name of the verification key flavor', 'ultra_honk')
+    .option('-r, --recursive', 'Whether a SNARK friendly key should be generated', false)
```

Comment:
```suggestion
    .requiredOption('-f, --flavor <string>', 'The name of the verification key flavor', 'ultra_honk')
    .option('-r, --recursive', 'Whether a SNARK friendly key should be generated', false)
```

Reply:
Not clear to me why this was removed.

Reply:
Sorry, accident.

Reply:
No worries, I don't think this is the cause of issues however as I don't think that the bb.js CLI is used in the failing tests.